### ISSUE TITLE
start qsynth or other sequencer on startup

### ISSUE THREAD

**Issue by matevz** [canorusmusic/canorus#112]
It would be useful to have an option to start `qsynth` on startup, if it's not running yet. I guess this would also mean it needs to wait a few seconds to get initialized.

**Comment by suamor** [comment]
I think this will be challenging, but doable in a similar way like the current print preview.
As we are on multiple platforms with multiple programs user should be able to define which ones to use for this purpose.
Otherwise we need to include all these external tools with Canorus. I doubt that qsynth is available on Android (not sure how midi is working here at all).

**Comment by matevz** [comment]
Yeah, I thought setting midi synth options (qsynth, timidity, something else?) would be in the Settings dialog as it is now for default typesetter and the print preview app.
